papis/commands/init.py

```diff
@@ -0,0 +1,87 @@
+r"""
+This command initializes a papis library interactively.
+
+Go to the directory where you want to have your papers
+
+::
+
+        mkdir -p ~/Documents/papers
+        cd ~/Documents/papers
+
+::
+
+        papis init
+
+and answer the questions.
+"""
+from typing import Optional  # noqa: F401
+import os
+
+import click
+
+import papis.utils
+import papis.config
+import papis.tui.utils
+import papis.logging
+
+logger = papis.logging.get_logger(__name__)
+
+INIT_PROMPTS = ["opentool", "editor", "use-git", "notes-name"]
```

Comment:
Not sure what else to put here, so I'll just list a few that don't sound too ridiculous (not advocating anything in particular): `database-backend`, `bibtex-unicode`, `ref-format`, `multiple-authors-separator`, `multiple-authors-format`, `citations-file-name`.

Maybe `ref-format` is worth adding, since different libraries could presumably have different preferences for that?

Reply:
I added most of these settings, I think they are meaningful, I also added a sentence to every setting to give more context to the user.

Now I noticed that in the future it would be nice if we could define the options and the documentation in the same file, then I would have been able to recycle the documentation of the settings here. But it is not super important.